packages/zipkin-transport-http/src/HttpLogger.js

```diff
@@ -60,12 +60,14 @@ class HttpLogger extends EventEmitter {
         headers: self.headers,
         timeout: self.timeout,
       }).then((response) => {
-        if (response.status !== 202) {
+        if (response.status !== 202 && response.status !== 200) {
           const err = 'Unexpected response while sending Zipkin data, status:' +
             `${response.status}, body: ${postBody}`;
 
           if (self.errorListenerSet) this.emit('error', new Error(err));
           else console.error(err);
+        } else {
+          this.emit('success', response);
```

Comment:
Couldn't find any way to test successful response handling on the logger's part without having some kind of positive feedback for the test to hook into.

Reply:
Adding an error handler and then calling `expect.fail()` in that was just making it raise unhandled promise errors and wouldn't actually fail.